components/atom/image/src/index.js

```diff
@@ -11,48 +11,54 @@ import {
   CLASS_IMAGE,
   CLASS_PLACEHOLDER,
   CLASS_SKELETON,
-  CLASS_SPINNER
+  CLASS_SPINNER,
+  DECODING,
+  FETCHPRIORITY,
+  LOADING
 } from './settings.js'
 import {htmlImgProps} from './types.js'
 
 const AtomImage = ({
-  placeholder,
-  skeleton,
+  alt,
   bgStyles,
-  spinner: Spinner,
+  decoding = DECODING.AUTO,
   errorIcon,
   errorText,
-  onError,
-  onLoad,
+  fetchpriority = FETCHPRIORITY.AUTO,
+  loading = LOADING.EAGER,
+  onError = () => {},
+  onLoad = () => {},
+  placeholder,
+  skeleton,
   sources = [],
-  alt,
+  spinner: Spinner,
   ...imgProps
 }) => {
   const imageRef = useRef()
-  const [loading, setLoading] = useState(true)
+  const [isLoading, setIsLoading] = useState(true)
   const [error, setError] = useState(false)
   const {src} = imgProps
 
   useEffect(() => {
-    setLoading(true)
+    setIsLoading(true)
     setError(false)
-  }, [src, setLoading, setError])
+  }, [src, setIsLoading, setError])
 
   const handleLoad = useCallback(() => {
     const loadCompleted = imageRef?.current?.complete
     if (loadCompleted === true) {
-      setLoading(!loadCompleted)
+      setIsLoading(!loadCompleted)
       onLoad && onLoad()
     }
-  }, [onLoad, setLoading])
+  }, [onLoad, setIsLoading])
 
   useEffect(() => {
     handleLoad()
   }, [handleLoad, imageRef])
 
   const classNames = cx(
     BASE_CLASS,
-    `is-${loading ? 'loading' : 'loaded'}`,
+    `is-${isLoading ? 'isLoading' : 'loaded'}`,
```

Comment:
I think this class should be **`.is-loading`** instead of .**`is-isLoading`**

Reply:
Is true!
I was use a find & replace 🙈

But we need to change the name, because the name `loading` we need to use with the HTML attribute